packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -1150,28 +1137,20 @@ export default decorate([
                               </Tooltip>{' '}
                               <input
                                 checked={offlineBackup}
+                                disabled={offlineSnapshot || checkpointSnapshot}
                                 onChange={effects.setOfflineBackup}
                                 type='checkbox'
                               />
                             </label>
                           </FormGroup>
                         </div>
                       )}
-                      {!state.offlineBackupActive && (
-                        <FormGroup>
-                          <label>
-                            <strong>{_('offlineSnapshot')}</strong>{' '}
-                            <Tooltip content={_('offlineSnapshotInfo')}>
-                              <Icon icon='info' />
-                            </Tooltip>{' '}
-                            <input
-                              checked={offlineSnapshot}
-                              onChange={effects.setOfflineSnapshot}
-                              type='checkbox'
-                            />
-                          </label>
-                        </FormGroup>
-                      )}
+                      <SnapshotModeSelect
+                        checkpointSnapshot={checkpointSnapshot}
```

Comment:
`value` insted of boolean props?

Reply:
To do this, i need to `construct/destruct` an object. And i don't see the utility of it.